.gitignore

```diff
@@ -0,0 +1,9 @@
+.DS_Store
+.idea
+*.log
+tmp/
+
+build/
+-build/conf
```

Comment:
not sure what the - in -build/conf means, but I think we need to hold on to build/conf (it has bblayers and local.conf in it)?

Reply:
or nevermind, think the -build/conf means we are holding on to the conf dir..